src/components/elements/Map-mapbox/utils.ts

```diff
@@ -366,7 +363,7 @@ export const addPopupToLayer = (
 };
 
 export const addSourceToLayer = (layer: any, map: mapboxgl.Map, polygonsData: Record<string, string[]> | undefined) => {
-  const { name, styles } = layer;
+  const { name, layerName, styles } = layer;
```

Comment:
It is not clear what is the difference between `layerName` and `name` is.
Please use another name for this, for example mapboxgl uses `id`, if you want to have a different purpose for this, as an example the geoserver name, please use that as `geoserverLayerName` but right now it is not obvious for someone with a glance at the code what is the difference.

Reply:
name changed